app/controllers/users_controller.rb

```diff
@@ -1,10 +1,20 @@
 class UsersController < ApplicationController
-  def show
-    @user = User.find_by id: params[:id]
-    return if @user
+  before_action :logged_in, only: %i(index edit update destroy)
```

Comment:
```suggestion
  before_action :logged_in, except: %i(show new create)
```
vì quá nửa action của restful nên em đổi`only` thành `except` dc nhé

Reply:
Dạ chị